lib/bracket-matcher.coffee

```diff
@@ -81,7 +82,7 @@ class BracketMatcher
       @editor.moveRight()
       false
     else if autoCompleteOpeningBracket
-      @editor.insertText(text + pair)
+      @origEditorInsertText(text + pair)
       @editor.moveLeft()
```

Comment:
On line 86. 

Before PR, the editor that `.moveLeft()` is the same as the editor that `.insertText()`.

Now they can potentially be two different editor. (If they are garantee to alwais the same I do not see the point of the PR). I'm not sure why a special treatment to the insertText method. There's a bunch of stuff done with the @editor object. Maybe it's that pointer that need to be protected against "unnecessary recursion" ?